geomstats/visualization.py

```diff
@@ -490,6 +494,186 @@ def draw(self, ax, **kwargs):
         ax.scatter(translation[:, 0], translation[:, 1], s=16, **kwargs)
 
 
+class KendallSphere:
+    """Class used to plot points in Kendall shape space of 2D triangles."""
+
+    def __init__(self, points=None, point_type='pre-shape'):
+        self.points = []
+        self.point_type = point_type
+        self.ax = None
+        self.elev, self.azim = None, None
+
+        # equilateral triangle (north pole)
```

Comment:
Instead/ in addition of putting the documentation as comments in the code, can you write it in the class docstring? This will allow it to be visible on the documentation website automatically. It looks like:

```
class KendallSphere:
    """Class used to plot points in Kendall shape space of 2D triangles.

    [Give an explanation of the Kendall sphere here]

    Attributes
    ----------
    points : list
        List of ...
    point_type : str
        Type of ...
    [...]
    ua : array-like, shape=...
        Tangent direction toward isocele triangle at vertex A
    etc
"""
```